common/protob/messages-management.proto

```diff
@@ -88,7 +88,13 @@ message Features {
     // optional bytes fw_vendor_keys = 26;      // obsoleted, use fw_vendor
     optional bool unfinished_backup = 27;       // report unfinished backup (equals to Storage.unfinished_backup)
     optional bool no_backup = 28;               // report no backup (equals to Storage.no_backup)
-    optional bool recovery_mode = 29;           // is recovery mode in progress
+    optional RecoveryStatus recovery_status = 29;  // whether or not we are in recovery mode and of what kind
+    enum RecoveryStatus {
+        NoRecovery = 0;
+        InNormalRecovery = 1;
+        InDryRunRecovery = 2;
+        InUnlockRepeatedBackupRecovery = 3;
+    }
```

Comment:
you need to separate the state of "recovering until we unlock the backup" and "we already unlocked the backup".
otherwise suite doesn't know if it should send RecoveryDevice or BackupDevice to continue

while you're at this please unsquish this and move both `RecoveryStatus` and `Capability` to the back of the message (like it is with `RecoveryKind`)

Reply:
https://github.com/trezor/trezor-firmware/pull/3743/commits/4c50fdc5e884c8f60c8b65e2daf5a77acdd4806e